docs/external/faq.md

```diff
@@ -88,6 +88,9 @@ e.g.
 systemctl restart docker
 ```
 
+#### I'm not able to create volume and I see "VolumeDriver.Create: Device not found" error
+
+From 0.19 release of vDVS, the plugin has to have a ESX service thats at least 0.19 or later. This applies for plugins on Linux platform. [#2023](https://github.com/vmware/docker-volume-vsphere/issues/2023)
```

Comment:
Let's not mention `Linux platform` explicitly, it should be OS independent.

/CC @govint  to confirm

Reply:
This issue will happen only with the plugin on Linux, but agree there is no need for the windows plugin to use an older VIB version. Its best that the users always upgrade both the pluigin and the VIB to matching levels.